src/requests/Analytic.ts

```diff
@@ -0,0 +1,23 @@
+import { AxiosRequestConfig } from 'axios';
+
+import { AnalyticOverviewParams } from '../models/params/AnalyticOverviewParams';
+
+/**
+ * @param analyticOverviewParams - The parameters to fetch the analytics overview.
+ *
+ * @public
+ */
+export function overview(analyticOverviewParams: AnalyticOverviewParams): AxiosRequestConfig {
```

Comment:
This is what I was talking about in my descriotion, Should we keep this at 1 object/param or divide it in multiple params?

Reply:
Since all other requests keep multiple params *(refer the the requests defined at `src/requests/User.ts`, it would be better to keep them as multiple instead of a single object for consistency reason. Later down the line, we may make it into an object. But keep it separate for now, removing the need of `AnalyticOverviewParams`.